aten/src/ATen/native/native_functions.yaml

```diff
@@ -4915,6 +4915,11 @@
     CPU: nested_from_padded_generic
     CUDA: nested_from_padded_cuda
 
+- func: _nested_tensor_size(Tensor self) -> Tensor
+  variants: method
```

Comment:
I know that christian was concerned about exposing the nested_size_tensor to users because the in-core method for tracking the size of tensor is not the same as the out of tree version. My understanding is that he didn't want any users to rely on the fact that nested tensor sizes is stored as a tensor since that may change in the future.

Reply:
@soulitzer